db/schema.rb

```diff
@@ -13,6 +13,53 @@
 
 ActiveRecord::Schema.define(:version => 20150701042203) do
 
+  create_table "credence_answers", :force => true do |t|
+    t.integer  "credence_question_generator_id"
+    t.text     "text"
+    t.text     "value"
+    t.datetime "created_at",                     :null => false
+    t.datetime "updated_at",                     :null => false
+    t.integer  "rank"
+  end
+
+  create_table "credence_games", :force => true do |t|
+    t.integer  "current_question_id"
+    t.integer  "score",               :default => 0,     :null => false
+    t.datetime "created_at",                             :null => false
+    t.datetime "updated_at",                             :null => false
+    t.integer  "user_id"
+    t.integer  "num_answered",        :default => 0,     :null => false
+    t.boolean  "debug",               :default => false, :null => false
```

Comment:
I don't love this. This is one of those things that people might scratch their heads over in a year's time wondering what it does. Can you either rename it to a more descriptive field name or remove it if it was only used during development?